docs/3.0.0-beta.x/guides/is-owner.md

```diff
@@ -0,0 +1,132 @@
+# Create is owner policy
```

Comment:
# Create is owner policy

This guide will explain how to restrict content edition to content authors only.

## Introduction

It is often required that the author of an entry is the only user allowed to edit or delete the entry.

This is a feature that is requested a lot and in this guide we will see how to implement it.

## Apply the author by default

When we are creating a new Article via `POST /articles` we will need to set the authenticated user as the author of the article.

To do so we will customize the `create` controller function of the Article API.

Reply:
DOne